redhat-operators/kie-cloud-operator/kiecloud-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiecloud-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
+    description: KieCloud can deploy RHPAM environments in the form of KieApp objects.
+    containerImage: quay.io/kiegroup/kie-cloud-operator:v0.1.0
```

Comment:
@tchughesiv is this image created by the OSBS build process and not an upstream image? The reason I am asking is to confirm that this needs to go into `redhat-operators` and not `community-operators`.

Reply:
If you are referring to the operator image itself... the ultimate goal is OSBS, but not there yet. that said, all of the rhpam images are in the RH registry.

Reply:
should we move this to community instead?

Reply:
I was referring to the operator image itself. I think it is best if this is slated as a community operator until you have the OSBS image ready.